res/values/strings.xml

```diff
@@ -59,14 +59,57 @@ This application contains no ads, doesn't make any network requests and is Open
   <string name="pref_character_size_summary">Size of characters displayed on the keyboard (%.2fx)</string>
   <string name="pref_theme">Theme</string>
   <string name="pref_theme_e_system">System settings</string>
-  <string name="pref_theme_e_dark">Dark</string>
-  <string name="pref_theme_e_light">Light</string>
-  <string name="pref_theme_e_black">Black</string>
+  <!-- THEMES_STRINGS_START -->
```

Comment:
The strings could also be outputted to the new file I suggest above but they need to be processed by `sync_translations.py`. It could be modified to also look at the new file when calculating the `baseline` (the English strings).